shared/src/vyxal/Elements.scala

```diff
@@ -2525,6 +2537,17 @@ object Elements:
         val temp =
           ListHelpers.transliterate(ListHelpers.makeIterable(a), b, c).mkString
         if VNum.NumRegex.matches(temp) then VNum(temp) else temp
+      case (a: VObject, b: String, c) => MiscHelpers.setObjectMember(a, b, c)
+      case (a: VObject, b: VList, c) =>
+        var obj = a.copy()
+        for i <- b.lst.indices do
```

Comment:
Why iterate over the indices and not the list itself?

Reply:
Because I need the indices

Reply:
Doesn't this work?
```scala
for elem <- b do
  obj = MiscHelpers.setObjectMember(obj, elem.toString, c)
```